pkg/lib/authn/otp/errors.go

```diff
@@ -0,0 +1,10 @@
+package otp
+
+import (
+	"github.com/authgear/authgear-server/pkg/api/apierrors"
+)
+
+var InvalidOTPCode = apierrors.Forbidden.WithReason("InvalidOTPCode")
+
+var ErrCodeNotFound = InvalidOTPCode.NewWithCause("otp code is expired or invalid", apierrors.StringCause("CodeNotFound"))
+var ErrInvalidCode = InvalidOTPCode.NewWithCause("invalid otp code", apierrors.StringCause("InvalidOTPCode"))
```

Comment:
We added new errors here. This implies new code returns new errors. Existing code checks old errors. `errors.Is` WILL NOT report the old ones and new ones equal. We have bugs here.

This is one refactoring technique I want to share.

> When we add new things, make sure we remove the old ones. 

This ensures existing code is 100% converted to use new code. If we do not do this, we could face mysterious bugs that take hours to diagnose.

---

- [ ] Remove pkg/lib/authn/authenticator/whatsapp/errors.go
- [ ] Remove pkg/lib/authn/authenticator/oob/errors.go
- [ ] We cannot simply remove pkg/lib/feature/verification/errors.go because if we search for "InvalidVerificationCode", we can see one usage in __error.html. To preserve the behavior, we have to translate the error. That is, we have to make ensure we still let __error.html see  the old errors.